app/helpers/application_helper.rb

```diff
@@ -2,4 +2,5 @@ module ApplicationHelper
   def full_title page_title
     base_title = "Sample App"
     page_title.blank? ? base_title : "#{page_title} | #{base_title}"
+
```

Comment:
you missing "end"

Reply:
i did put the end but it did not appear on the git